src/Sylius/Behat/Context/Setup/ProductContext.php

```diff
@@ -675,6 +675,14 @@ public function thisProductBelongsToShippingCategory(ProductInterface $product,
     }
 
     /**
+     * @When administrator change shipping category to :shippingCategory for product :product
```

Comment:
administrator change**s**

or even

product "Westworld host" has/gets its shipping category changed to "Small stuff"